src/components/Reactions/ReactionsListModal.tsx

```diff
@@ -16,13 +17,21 @@ type ReactionsListModalProps<
     reactions: ReactionSummary[];
     selectedReactionType: string | null;
     onSelectedReactionTypeChange?: (reactionType: string) => void;
+    sortReactionDetails?: ReactionDetailsComparator;
   };
 
+const defaultSortReactionDetails: ReactionDetailsComparator = (a, b) => {
+  const aName = a.user?.name ?? a.user?.id;
+  const bName = b.user?.name ?? b.user?.id;
+  return aName ? (bName ? aName.localeCompare(bName, 'en') : 1) : -1;
```

Comment:
Do I understand it well that if `bName` is `undefined, then the resulting order will be `[b,a]` because the return value is > 0?

Reply:
Yes. But really, that's never going to happen, since we fallback to `user.id`, and `user`, although marked optional, is always defined.

Just a precaution, I've changed the order so that users with undefined names go to the bottom of the list, instead of to the top.